README.md

```diff
@@ -103,6 +104,105 @@ number of consenting nodes than PBFT.
 For more detailed description of the protocol, refer to [Efficient
 Byzantine Fault-Tolerance paper][minbft-paper].
 
+## Quick Start ##
+
+This quick start shows you how to runnning this project using Docker
+containers. This is the easiest way to try out this project with minimal
+setup. If you want to build and run without Docker, skip this section.
```

Comment:
Actually I like putting this new section subtree just before "Roadmap" section and inserting short notes as done in the previous section. But this might conflict with Sergey's suggestion, so I'm fine if you decide to go with this.